.travis.yml

```diff
@@ -24,6 +24,19 @@ matrix:
           sudo: required
         - rust: nightly
           env: TASK=clippy
+        - language: python
+          install: pip3 install -r tests/client-dbus/requirements.txt
+          before_script:
+              - cd tests/client-dbus
+          env: TASK=lint
+          before_install:
+              - sudo apt-get -qq update
+              - sudo apt-get install -y libdbus-glib-1-dev
+          language: python
+          python: "3.4"
```

Comment:
This is probably behind the times. Even my version of Fedora has Python 3.6 now. So probably best to update it or find a way to generically specify Python 3.